plugins/repository-gcs/src/internalClusterTest/java/org/elasticsearch/repositories/gcs/GoogleCloudStorageBlobStoreRepositoryTests.java

```diff
@@ -312,11 +312,14 @@ public void maybeTrack(final String request, Headers requestHeaders) {
             if (Regex.simpleMatch("GET /storage/v1/b/*/o/*", request)) {
                 trackRequest("GetObject");
             } else if (Regex.simpleMatch("GET /storage/v1/b/*/o*", request)) {
-                trackRequest("ListObjects");
+                trackRequest("ListObjGects");
```

Comment:
This is the wrong name? How is it passing tests?

Reply:
Good catch! The test was expecting some predefined keys and using `getOrDefault(key, 0)` 🤦  instead of just comparing the mock and `RepositoryStats` maps. I've changed it, it should catch those failures in the future.